tests/appsec/integrations/pygoat_tests/Dockerfile.pygoat.2.0.1

```diff
@@ -0,0 +1,40 @@
+FROM python:3.11.0b1-buster
+
+# set work directory
+WORKDIR /app
+
+
+# dependencies for psycopg2
+RUN apt-get update && apt-get install --no-install-recommends -y dnsutils git libpq-dev=11.16-0+deb10u1 python3-dev=3.7.3-1 \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+
+# Set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# Download pygoat
+RUN git clone https://github.com/adeyosemanputra/pygoat.git
+
+
+# copy project
+COPY . /app/
+
+# copy and install ddtrace
+WORKDIR /
+COPY ../../../../ ddtrace
+RUN pip install --no-cache-dir ddtrace
+
+WORKDIR /app/pygoat
+# Install dependencies
+RUN git checkout v2.0.1
```

Comment:
nit: I'd move this up to the "download" section (or move the cloning down to here).